sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -117,6 +124,20 @@ export namespace ConnectionContext {
       audience: options.managementSessionAudience
     };
     connectionContext.managementSession = new ManagementClient(connectionContext, mOptions);
+    connectionContext.isConnectionClosing = function() {
+      // When the connection is not open, but the remote end is open,
+      // then the rhea connection is in the process of terminating.
+      return Boolean(!this.connection.isOpen() && this.connection.isRemoteOpen());
+    };
+    connectionContext.isDisconnecting = false;
```

Comment:
Can we move this up to right after the connection context was created to group all the initialization of simple types in the same place?